src/data/repositories/consts/DiseaseOutbreakConstants.ts

```diff
@@ -0,0 +1,140 @@
+    appropriateCaseManagementDate: "RTSL_ZEB_TEA_SPECIFY_DATE2",
+    initiatePublicHealthCounterMeasuresNA: "RTSL_ZEB_TEA_APPROPRIATE_PUBLIC_HEALTH",
+    initiatePublicHealthCounterMeasuresDate: "RTSL_ZEB_TEA_SPECIFY_DATE3",
+    initiateRiskCommunicationNA: "RTSL_ZEB_TEA_APPROPRIATE_RISK_COMMUNICATION",
+    initiateRiskCommunicationDate: "RTSL_ZEB_TEA_SPECIFY_DATE4",
+    establishCoordination: "RTSL_ZEB_TEA_ESTABLISH_COORDINATION_MECHANISM",
+    responseNarrative: "RTSL_ZEB_TEA_RESPONSE_NARRATIVE",
+    incidentManager: "RTSL_ZEB_TEA_ASSIGN_INCIDENT_MANAGER",
+    notes: "RTSL_ZEB_TEA_NOTES",
+} as const;
+
+export type DiseaseOutbreakCode = GetValue<typeof DiseaseOutbreakCodes>;
+
+export type KeyCode = (typeof DiseaseOutbreakCodes)[keyof typeof DiseaseOutbreakCodes];
+
+export function isStringInDiseaseOutbreakCodes(code: string): code is KeyCode {
+    return Object.values(DiseaseOutbreakCodes).includes(code as KeyCode);
+}
+
+export function isHazardType(hazardType: string): hazardType is HazardType {
+    return [
+        "Biological:Animal",
+        "Biological:Human",
+        "Chemical",
+        "Environmental",
+        "Unknown",
+    ].includes(hazardType);
+}
+
+export function getValueFromDiseaseOutbreak(
+    diseaseOutbreak: DiseaseOutbreakEventBaseAttrs
+): Record<DiseaseOutbreakCode, string> {
+    return {
+        RTSL_ZEB_TEA_EVENT_id: diseaseOutbreak.eventId.toString(),
+        RTSL_ZEB_TEA_EVENT_NAME: diseaseOutbreak.name,
+        RTSL_ZEB_TEA_HAZARD_TYPE: getHazardTypeCode(diseaseOutbreak.hazardType),
+        RTSL_ZEB_TEA_MAIN_SYNDROME: diseaseOutbreak.mainSyndromeCode,
+        RTSL_ZEB_TEA_SUSPECTED_DISEASE: diseaseOutbreak.suspectedDiseaseCode,
+        RTSL_ZEB_TEA_NOTIFICATION_SOURCE: diseaseOutbreak.notificationSourceCode,
+        RTSL_ZEB_TEA_AREAS_AFFECTED_PROVINCES: getOUTextFromList(
+            diseaseOutbreak.areasAffectedProvinceIds
+        ),
+        RTSL_ZEB_TEA_AREAS_AFFECTED_DISTRICTS: getOUTextFromList(
+            diseaseOutbreak.areasAffectedDistrictIds
+        ),
+        RTSL_ZEB_TEA_INCIDENT_STATUS: diseaseOutbreak.incidentStatus,
+        RTSL_ZEB_TEA_DATE_EMERGED: diseaseOutbreak.emerged.date.toISOString(),
+        RTSL_ZEB_TEA_DATE_EMERGED_NARRATIVE: diseaseOutbreak.emerged.narrative,
+        RTSL_ZEB_TEA_DATE_DETECTED: diseaseOutbreak.detected.date.toISOString(),
+        RTSL_ZEB_TEA_DATE_DETECTED_NARRATIVE: diseaseOutbreak.detected.narrative,
+        RTSL_ZEB_TEA_DATE_NOTIFIED: diseaseOutbreak.notified.date.toISOString(),
+        RTSL_ZEB_TEA_DATE_NOTIFIED_NARRATIVE: diseaseOutbreak.notified.narrative,
+        RTSL_ZEB_TEA_INITIATE_INVESTIGATION:
+            diseaseOutbreak.earlyResponseActions.initiateInvestigation.toISOString(),
+        RTSL_ZEB_TEA_CONDUCT_EPIDEMIOLOGICAL_ANALYSIS:
+            diseaseOutbreak.earlyResponseActions.conductEpidemiologicalAnalysis.toISOString(),
+        RTSL_ZEB_TEA_LABORATORY_CONFIRMATION: diseaseOutbreak.earlyResponseActions
```

Comment:
For NA values is the other way arround:

```
RTSL_ZEB_TEA_LABORATORY_CONFIRMATION: diseaseOutbreak.earlyResponseActions
            .laboratoryConfirmation.na
            ? ""
            : "true"
```

Reply:
Hi @anagperal In DHIS all of these are mandatory tracked entity attribute of Yes only type. They cannot be null or empty. The app implementation needs to take care of the true value, we can discuss further over call.